monai/transforms/spatial/array.py

```diff
@@ -441,14 +441,16 @@ class Rotate(Transform, ThreadUnsafe):
             the output data type is always ``np.float32``.
     """
 
+    backend = [TransformBackends.TORCH]
```

Comment:
Could you please help add the numpy conversion as we discussed in https://github.com/Project-MONAI/MONAI/issues/2933?

Thanks.

Reply:
Sorry, what do you want me to do? When backend is TORCH, numpy->torch conversion will be done if necessary.

Reply:
Sorry maybe I didn't make it clear, I mean we should support the case `numpy in` -> `numpy out` to be compatible with old transform.

Thanks.

Reply:
Hm, would it be acceptable if I modified all "old" transforms to convert to numpy at the start of the transform? This will ensure backwards compatibility.

Reply:
I think @Nic-Ma means we return the original type like what we do here, converting the `zoomed` type to `img` type: https://github.com/Project-MONAI/MONAI/blob/c9b08204b7844b74258348fd74b2b8117912a57e/monai/transforms/spatial/array.py#L644-L645
we can merge all these essential PRs first and review everything for compatibility.